src/types/types.ts

```diff
@@ -243,9 +246,9 @@ export enum CallAlertType {
 export type CallAlertConfig = {
     type: CallAlertType;
     icon: string;
-    bannerText: string;
-    tooltipText: string;
-    tooltipSubtext: string;
+    bannerText: MessageDescriptor;
```

Comment:
I didn't realize we shared all these types. Does it mean we can get rid of them on the web side once we start using this?

Reply:
Yep, that's what I'm working on in the webapp's PR.

Reply:
Actually, we're not sharing _all_ of them... I moved all of the webapp's types here. Maybe that's going to be annoying for just the types only used by webapp...

Reply:
Alright, not opposed to this at all but I could see a bit of pain when we need to add types that are clearly meant for one side only since we'd need to send a PR here every time. Would it make sense to keep here only those that are actually to be shared?

Reply:
Yah, that's what I was just rethinking. Let me work on that a bit. Need to go through each one and separate them. I just did a quick and dirty move, and that's probably going to cause annoyances later.

Reply:
That would be great, thanks.